Packs/AzureKeyVault/Integrations/AzureKeyVault/README.md

```diff
@@ -0,0 +1,1232 @@
+Use Key Vault to safeguard and manage cryptographic keys and secrets used by cloud applications and services.
+This integration was integrated and tested with version xx of AzureKeyVault
```

Comment:
please change the xx to the version you actually used

Reply:
Done